layouts/shortcodes/cloakemail.html

```diff
@@ -1,4 +1,12 @@
-{{- $parts := split (index .Params 0) "@" -}}
+<!-- use mailto: if href is not set -->
```

Comment:
Could you explain the rationale of having other protocols? I can't see the use case. TIA.

Reply:
[I for example](https://mehl.mx/contact/) have xmpp as another protocol, but I could also think of tel (telephone, SIP), Telegram or WhatsApp

Reply:
I understand now. Good idea indeed.